packages/create-mc-app/src/constants.ts

```diff
@@ -8,4 +8,13 @@ const availableTemplates = {
   'starter-typescript': 'starter-typescript',
 } as const;
 
-export { applicationTypes, availableTemplates };
+const supportedCloudRegions = {
```

Comment:
Is this the correct file to keep the supportedCloudRegions object or should I keep it in the throwIfCloudRegionNotSupported function?

Reply:
It seems this object is already exported from the `@commercetools-frontend/application-config`. Perhaps we can add that as a dependency and import it from there.
Unless others think it doesn't make sense to make the package bundle 296 kB bigger.

Reply:
That's fine. Thanks

Reply:
azure-us has recently been added as a new cloud identifier. Should I include this identifier as well?

This is the object that has been exported from `@commercetools-frontend/application-config`

```
export const CLOUD_IDENTIFIERS = {
  GCP_AU: 'gcp-au',
  GCP_EU: 'gcp-eu',
  GCP_US: 'gcp-us',
  AWS_FRA: 'aws-fra',
  AWS_OHIO: 'aws-ohio',
  AWS_CN: 'aws-cn',
  AZURE_US: 'azure-us',
} as const; 
```
However, azure-us is not currently present in the [documentation](https://docs.commercetools.com/merchant-center-customizations/api-reference/custom-application-config#cloudidentifier) for Custom Application Config and Custom View Config. 

Is azure-us supported as a cloud-identifier for the custom-applications and custom views? If it's supported then we might have to update the documentation here as well.

Reply:
> Is azure-us supported as a cloud-identifier for the custom-applications and custom views? 

It is, [here's the mapping](https://github.com/commercetools/merchant-center-application-kit/blob/67a35e2046ac76dfbee7a2d93bb21595c6d4f086/packages/application-config/src/utils.ts#L5-L32).

> If it's supported then we might have to update the documentation here as well.

Thanks, that's a good point 👍 It will be covered by the docs team upon azure launch, I can point you to a relevant PR if you need

Reply:
The following deprecation warning shows up after importing CLOUD_IDENTIFIERS from `@commercetools-frontend/application-config`. Is it a good idea to import this object from application-config or should I declare it in the constants file instead?

![Screenshot 2024-07-04 at 14 32 11](https://github.com/commercetools/merchant-center-application-kit/assets/173251319/f17c0ead-801c-4dda-9058-6c32e57d3df6)

Reply:
The deprecation is unrelated, it's a node.js thing.

No worries 😉

Reply:
Updated in this PR: [1dc901c](https://github.com/commercetools/merchant-center-application-kit/pull/3564/commits/1dc901c3d497af7cc5c144f61f234ae3a4e1f0c3)